fedora-manual-setup.md

```diff
@@ -35,36 +35,40 @@ sudo dnf install sphinx chromedriver postgresql-devel mysql-devel ImageMagick  I
 
 ### Database
 
-The application requires a database that can either be [PostgreSQL](https://www.postgresql.org), [MySQL](https://www.mysql.com) or [Oracle database](https://www.oracle.com/database/). We recommend running MySQL in a [Podman](https://podman.io/) container:
+The application requires a database that can either be [PostgreSQL](https://www.postgresql.org), [MySQL](https://www.mysql.com) or [Oracle database](https://www.oracle.com/database/). MySQL will be used by default.
+
+###### MySQL
+
+We recommend running it in a [Podman](https://podman.io/) container:
 
 ```sh
 podman run -d -p 3306:3306 -e MYSQL_ALLOW_EMPTY_PASSWORD=true --name mysql57 mysql:5.7
 ```
 
 ### Redis
 
-[Redis](https://redis.io) is an in-memory data store used as DB for some of the data and it has to be running for the application to work. We recommend running Redis in a [Podman](https://podman.io/) container:
+[Redis](https://redis.io) is an in-memory data store used as DB for some of the data and it has to be running for the application to work. We recommend running it in a [Podman](https://podman.io/) container:
 
 ```
 podman run -d -p 6379:6379 redis
 ```
 
-Alternatively, it can be run directly on your machine with `dnf`:
+Alternatively, Redis can be run directly on your machine with `dnf`:
 
 ```
 sudo dnf install redis
 sudo systemctl restart redis
 ```
 
-### Rails cache (Memcached)
+### Memcached
 
-If available, Rails and Redis will use [Memcached](https://www.memcached.org) for caching. Installing it is completely optional but still recommended. We recommend running memcached in a [Podman](https://podman.io/) container:
+If available, Rails and Redis will use [Memcached](https://www.memcached.org) for caching. Installing it is completely optional but still recommended. We recommend running it in a [Podman](https://podman.io/) container:
```

Comment:
I don't think Redis can use memcached for anything. Rather memcached according to some sources can be used instead of memcached, although Rails does not have such integration afaik.

Reply:
Memcached can be used instead of _redis_ you mean? What do we need memcached for then? I haven't installed it and never used it AFAIK.

Reply:
..Redis can be used for the use cases memcached is used for..